test/node/resendable-call-from-client.js

```diff
@@ -109,3 +112,12 @@ function reconnect(port) {
     });
   });
 }
+
+let calledClose = 0;
+function close() {
+  if (++calledClose !== 2) {
```

Comment:
Subjective nit (take it or ignore it): I'm not very fond of such expressions and I'd rather split it into two distinct statements.